dlgr/griduniverse/experiment.py

```diff
@@ -1033,7 +1033,7 @@ def setup(self):
         self.node_by_player_id = {}
 
     def recruit(self):
-        self.recruiter.close_recruitment()
+        self.recruiter().close_recruitment()
```

Comment:
I don't think this is correct. `self.recruiter.close_recruitment()` works for me and there's no uses of `recruiter()` as far as I can tell?

Reply:
If we're targeting `dallinger/master` then `recruiter()` is still a method call. It's converted to a property on `dallinger/development`.

Reply:
Ending the HIT was failing with a TypeError without using the method call on the first call without this change (it would succeed on subsequent calls though).